packages/@lwc/integration-karma/test/static-content/index.spec.js

```diff
@@ -396,3 +398,45 @@ describe('event listeners on deep paths', () => {
         }
     });
 });
+
+describe('comments', () => {
```

Comment:
Not sure if this top-level `describe` adds value. We can remove it in favor of top-level `it` if you agree.

Reply:
@ekashida I added a more [descriptive text](https://github.com/salesforce/lwc/pull/3934/files#diff-c1197d00755cf941e7a000d5b59dad82bcf7e4133c7445dd7e792893a6f7a49aR402) for the test but kept it in a `describe` block to keep it in line with what we're doing with the rest of the tests in this file.